secret-store/secret_store.py

```diff
@@ -0,0 +1,145 @@
+from dataclasses import dataclass
+from sqlcipher3 import dbapi2 as sqlcipher
+import logging
+import os
+from typing import Optional, Union
+
+
+_logger = logging.getLogger(__name__)
+# logging.basicConfig(level=logging.INFO, format="%(message)s")
+
+
+@dataclass(frozen=True)
+class Table:
+    name: str
+    columns: list[str]
+
+
+SECRETS_TABLE = Table("secrets", ["user", "password"])
+CONFIG_ITEMS_TABLE = Table("config_items", ["item"])
+
+
+@dataclass(frozen=True)
+class Credentials:
+    user: str
+    password: str
+
+
+class Secrets:
```

Comment:
Would you be up for a pairing review on this class?

Reply:
Sure!